pyOCD/board/mbed_board.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, link, target=None, frequency=1000000):
         """
         self.native_target = None
         self.test_binary = None
+        link.set_deferred_transfer(True)
```

Comment:
Thought about that, but MbedBoard is specifically tied to pyDAPAccess, while Board is not really.